src/Stepper/Step.js

```diff
@@ -0,0 +1,95 @@
+import React, {Component, PropTypes} from 'react';
+
+const getStyles = ({index}, {stepper}) => {
+  const {orientation} = stepper;
+  const styles = {
+    root: {
+      flex: '0 0 auto',
+    },
+  };
+
+  if (index > 0) {
+    if (orientation === 'horizontal') {
+      styles.root.marginLeft = -6;
+    } else if (orientation === 'vertical') {
+      styles.root.marginTop = -14;
+    }
+  }
+
+  return styles;
+};
+
+export default class Step extends Component {
+
+  static propTypes = {
+    /**
+     * Sets the step as active. Is passed to child components.
+     */
+    active: PropTypes.bool,
+    /**
```

Comment:
Ok, I'll leave for now! Was just force of habit -- makes me wonder if it's worth a tiny `eslint` rule if we prefer one way over another.